st2reactor/st2reactor/sensor/compat.py

```diff
@@ -1,73 +0,0 @@
-# Licensed to the StackStorm, Inc ('StackStorm') under one or more
```

Comment:
That's unrelated to this PR, I just noticed this file is not actually used and obsolete.